frontend/public/components/utils/timestamp.tsx

```diff
@@ -54,7 +55,11 @@ export const Timestamp = connect(nowStateToProps)((props: TimestampProps) => {
       <Tooltip
         content={[
           <span className="co-nowrap" key="co-timestamp">
-            {mdate.toISOString()}
+            {t('{{date, MMM D, h:mm a z}}', {
```

Comment:
I'm tempted to say we should just use

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Intl/DateTimeFormat/DateTimeFormat

And not rely on moment.js for dates. The problem is that Safari doesn't support date/time formats.

Not something to tackle in this PR since we are already using moment.js.